src/Connection.c

```diff
@@ -593,13 +593,13 @@ static int Connection_Init(udt_Connection *self, PyObject *args,
     if (dpiContext_initConnCreateParams(g_DpiContext, &dpiCreateParams) < 0)
         return Error_RaiseAndReturnInt();
     if (!PyArg_ParseTupleAndKeywords(args, keywordArgs,
-            "|OOOikO!OOOiOssOOOO", keywordList, &usernameObj, &passwordObj,
+            "|OOOikO!OOOiOssOOOOI", keywordList, &usernameObj, &passwordObj,
```

Comment:
Are you sure this should be `I` and not `i`? I'm not about to dig through the documentation to check, but I see `i` used elsewhere for integral (I think?) types so I'm not sure if there's a difference.